src/koaApi.coffee

```diff
@@ -77,10 +79,27 @@ exports.setupApp = (done) ->
   app.use route.get '/visualizer/events/:receivedTime', visualizer.getLatestEvents
   app.use route.get '/visualizer/sync', visualizer.sync
 
-  app.use route.get '/metrics', metrics.getGlobalLoadTimeMetrics
-  app.use route.get '/metrics/status', metrics.getGlobalStatusMetrics
-  app.use route.get '/metrics/:type/:channelId', metrics.getChannelMetrics
-  
+# -- Old metrics Routes Purposely commented out for Backward Compatibility
+  app.use route.get '/mongometrics', metrics.getGlobalLoadTimeMetrics
+  app.use route.get '/mongometrics/status', metrics.getGlobalStatusMetrics
+  app.use route.get '/mongometrics/:type/:channelId', metrics.getChannelMetrics
```

Comment:
These tests fail because they inject data directly into the mongo database without running actual transactions, therefor, the metrics are never sent to the statsd server.